boards/arm/rcar_spider/doc/rcar_spider.rst

```diff
@@ -0,0 +1,187 @@
+.. _rcar_spider_boards:
+
+Renesas R-Car Spider
+####################
+
+Overview
+********
+
+R-Car S4 enables the launch of Car Server/CoGW with high performance, high-speed networking, high security and high functional safety levels that are required as E/E architectures evolve into domains and zones.
```

Comment:
can you wrap the lines to either 80 or 100 columns? sorry we don't have a check for this yet but that's how these docs are normally done

Reply:
Ok, but how should I manage links that are longer than 100 characters then ?
They are gathered at the bottom of the rst file

Reply:
those are fine, just wrap the normal text